{{ cookiecutter.project_name_dashed }}/.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: ❓ Question
+description: Ask a question about this project
+title: "[Ques]: "
+labels: ["question"]
+body:
+  - type: textarea
+    id: question
+    attributes:
+      label: Your Question
+      description: What would you like to know?
+    validations:
+      required: true
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: First Check
+      options:
+        - label: I have searched existing issues and discussions
+          required: true
```

Comment:
I think we can remove that, it is already implied (and more specifically) by the third point. Also, I think the first check should be for the documentation, _then_ the issues.